geomesa-blobstore/README.md

```diff
@@ -17,3 +17,40 @@ The class `WKTFileHandler` serves as an example of that latter approach.
 
 `FileHandler` implementations are loaded at runtime by the Java ServiceLoader.  As such, available 
 implementations should have their classname registered in `META-INF/services/org.locationtech.geomesa.blob.core.handlers.FileHandler`.
+
+## REST api
+
+The RESTful api for the Blobstore can be easily utilized via cURL as so:
+
+Before we can do anything we must first register a blob store. This typically only needs to be run once in the lifetime
+of the webserver, so if the server is shutdown or restarted the registration call will need to be repeated.
```

Comment:
There is a ticket for it. The slight problem is that ideally it would use the geoserver_data_dir, which could introduce license issues. However, the solution has already been implemented in e.g. geomesa/geomesa-web/geomesa-web-data/src/main/resources/applicationContext.xml.